R/adjust.R

```diff
@@ -75,12 +75,8 @@ adjust <- function(data,
                    ignore_case = FALSE,
                    regex = FALSE,
                    verbose = FALSE) {
-  if (!all(colnames(data) == make.names(colnames(data), unique = TRUE))) {
-    insight::format_warning(
-      "Bad column names (e.g., with spaces) have been detected which might create issues in many functions.",
-      "Please fix it (you can run `names(mydata) <- make.names(names(mydata))` for a quick fix)."
-    )
-  }
+  # make sure column names are syntactically valid
+  .check_dataframe_names(data, action = "error")
```

Comment:
Should we error here? This is not really wrong, it's "just" bad practice. I think this could break a bunch of code (could be done for 1.0 though).

I think we could also recommend `janitor::clean_names()`.

Reply:
`janitor::clean_names()` is included in the message. I think an error here is ok, because `adjust()` fails anyway with invalid column names, so why show a warning and an additional error?

Reply:
Right, my bad. I didn't know `adjust()` would fail with invalid column names so erroring is fine.